Non-Interactive-Protocol.md

```diff
@@ -0,0 +1,173 @@
+		- MUST spend the change output with a CPFP increasing package feerate to confirm soonly
+		- MUST start a bumping timer of length `congestion_delay`
+		- if `congestion_delay` expires:
+			- MUST spend the change output with a CPFP increasing package feerate to confirm soonly
+			- MUST replace according to BIP 125
+			- MUST reschedule timer for next bump
+		- if `contract_maturity_bound` is 20 blocks ahead from local tip:
+			- MUST spend the funding input with a higher-feerate transaction/absolute fee than any previously propagated package feerate
+			- MUST start a bumping timer of length `security_delay`
+			- MUST disable `congestion_delay` timer
+		- if `security_delay` expires:
+			- MUST spend the funding input with a higher-feerate transaction/absolute fee than any previously propagated package feerate
+			- MUST replace according to BIP 125
+			- MUST reschedule timer for next bump
+		- MUST NOT spend more in bumping fees than its collateral input value
+
+A node:
+	- if the funding transaction confirms:
+		* MUST track spend of the CET output
+		* MUST advances its off-chain state to the execution phase
+		* MUST note its change output for latter spend
+
+A node:
+	- if the collateral double-spend confirms:
+		* MUST abort the DLC and prevent further interactions with counterparty
+
+## Rationale
+
+Funding transaction must be confirmed to consider the off-chain state initialized. As mempools
+congestion may fluctuate between fees negotiation and transaction propagation, party must adjust
+unilaterally feerate according to their confirmation preferences. A party may choose to wait
+for mempools draining before to adjust the feerate to avoid a fee waste as block height of 
+transition to execution phase doesn't matter, assuming it's before `contract_maturity_bound`.
+
+Reaching near `contract_maturiy_bound` requires to confirm the funding transaction quickly as it's
+now a security matter. After oracle signature release, a malicious party could try to double-spend
+an unconfirmed funding transaction to avoid loosing its collateral.
+
+
+# Execution Phase
+
+The execution phase transitions to the terminal phase either by a CET or a refund
+transaction confirmations.
+
+## Requirements
+
+A node:
+	- if an oracle signature is received:
+		- MUST finalize the CET with a local `SIGHASH_ALL` signature
+		- MUST broadcast the CET
+		- MAY spend its outcome output with a CPFP increasing package feerate to confirm soonly
+		- if the CET is not confirmed and `contract_timeout is 20 blocks ahead of local tip:
+			- MUST rebroadcast the CET
+			- MUST spend its outcome output with a CPFP increasing package feerate to confirm soonly
+			- MUST start a bumping timer of length `security_delay`
+		- if `security_delay` expires:
+			- MUST spend the change output with a replacing CPFP increasing package feerate to confirm soonly
+			- MUST replace according to BIP 125
```

Comment:
Replace what with what? Are you talking about the transaction spending the CET output?

Reply:
Made it clearer replacing the CPFP transaction. There is non-null risk that an ulterior rebroadcasting of higher-feerate CPFP might be rejected by the mempool if you add unconfirmed inputs.